docs/tutorials/serving/rest_simple.ipynb

```diff
@@ -355,7 +355,12 @@
       },
       "outputs": [],
       "source": [
-        "!{SUDO_IF_NEEDED} apt-get install tensorflow-model-server"
+        "#!{SUDO_IF_NEEDED} apt-get install tensorflow-model-server #installing Tensorflow Model server 2.8 instead of latest version\n",
+        "Tensorflow Serving >2.9.0 required `GLIBC_2.29` and `GLIBCXX_3.4.26`. Currently colab environment doesn't support latest version of`GLIBC`,\n",
+        "so workaround is to use specific version of Tensorflow Serving `2.8.0` to mitigate issue.\n",
```

Comment:
Because line 359 and 360 is in a different line, it is not interpreted as a comment. So this cell will fail... Please